### PR TITLE
Add Support Level heading and table to guides

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Issue 420

### CODE REVIEW

File: docs/getting-started-guides/clc.md [hunk 1]
```diff
@@ -332,3 +332,11 @@ These are the known items that don't work on CenturyLink cloud but do work on ot
 
 If you want more information about our Ansible files, please [read this file](https://github.com/CenturyLinkCloud/adm-kubernetes-on-clc/blob/master/ansible/README.md)
 
+## Further reading
+
+Please see the [Kubernetes docs](/docs/) for more details on administering
+and using a Kubernetes cluster.
+
+For support level information, see the [Table of solutions](/docs/getting-started-guides/#table-of-solutions) chart.
```
Comment (johndmulhausen) [inline docs/getting-started-guides/clc.md]: Why no inline table here?

**Comment by johndmulhausen** [comment]
@bgrant0607 @janetkuo @pwittrock 
This is staged at jamesdefabia.github.io and is from a Write the Docs contributor. To see it, scroll to the bottom of http://jamesdefabia.github.io/docs/getting-started-guides/centos/centos_manual_config/
It's meant to show the support level for various solutions (and who to contact for help). Currently that is all in a table at: http://jamesdefabia.github.io//docs/getting-started-guides/#table-of-solutions which is totally disjointed from the information. 

@JamesDeFabia -- You need to be sure there is always a blank line above the headings, otherwise they don't render as headings. For example, see ##Support Level on this page: https://github.com/JamesDeFabia/jamesdefabia.github.io/edit/master/docs/getting-started-guides/aws.md

And how it renders: 
http://jamesdefabia.github.io/docs/getting-started-guides/aws/

I sent a PR that fixes it in aws.md as an example fix, but please check the rest.

**Comment by JamesDeFabia** [comment]
@johndmulhausen  I edited aws.md and committed the change. I'll make the other edits on Thursday when I get home from conference.

File: docs/getting-started-guides/clc.md (hunk 1 above)
Comment (JamesDeFabia) [inline docs/getting-started-guides/clc.md]: On this one , I don't know which row to use

**Comment by johndmulhausen** [comment]
Thanks, @JamesDeFabia ! Could you email me at johndylan@google.com so I can send your GCP credit? :)

**Pull request merged**